### ISSUE TITLE
Aliasing colors

### ISSUE THREAD

**Issue by ReinhardtR** [brattonross/windy-radix-palette#7]
This plugin is awesome! But it's missing one thing for me: aliasing color names.

I would like to be able to alias my colors just like described in the radix guide: https://www.radix-ui.com/docs/colors/getting-started/aliasing

This is what I was imagining, but it may not be possible with how the current API works. 🤷‍♂️  
```js
const radixColors = require("@radix-ui/colors");

// tailwind config
{ 
  ...
  plugins: [
    require("windy-radix-palette")({
      colors: {
        gray: radixColors.slate
      },
    }),
  ],
}
```

**Comment by ReinhardtR** [comment]
This seems to work, so I will do this for now.

```js
require("windy-radix-palette")({
  colors: {
    slate: radixColors.slate,
    gray: {
      gray1: "var(--slate1)",
      ...
    },
  },
}),
```

**Comment by brattonross** [comment]
Using CSS variables is the way that I would recommend for now 👍

I would do this by extending tailwind theme colors like so:

```js
module.exports = {
  theme: {
    extend: {
      colors: {
        gray: {
          1: "var(--slate1)",
          2: "var(--slate2)",
          // ...
        }
      }
    }
  },
  // ...
};
```

Writing out the 12 variables for each color you want to alias can be tedious, so I'd recommend creating a function to help:

```js
const toRadixVars = (color) =>
  [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12].reduce(
    (obj, n) => ({
      ...obj,
      [n]: `var(--${color}${n})`,
    }),
    {}
  );
```

I find myself writing this function in every project where I use this plugin, so I may add a couple of functions to this package to prevent having to rewrite them every time&ndash;one for the case where you want to alias an entire scale, and another for when you want to alias a single color.

This doesn't allow you to use [mutable aliases](https://www.radix-ui.com/docs/colors/getting-started/aliasing#mutable-aliases) however.

**Comment by brattonross** [comment]
I've released a new version (0.4.0), which includes a couple of functions to assist with aliasing colors. You can see an explanation [in the docs](https://windy-radix-palette.vercel.app/docs/palette/aliasing).

I hope this helps!
